src/user-icon/user-icon.scss

```diff
@@ -136,60 +136,91 @@
       color: t(iui-text-color-muted);
     }
   }
-}
-
-@mixin iui-user-icon-list {
-  display: flex;
 
-  .iui-user-icon:not(:first-child) {
-    margin-left: -$iui-xs;
+  &.iui-user-icon-count:focus {
+    @include themed {
+      box-shadow: t(iui-focus-box-shadow);}
+    border-radius: 50%;
 
-    .iui-initials {
+    &.iui-user-icon > .iui-stroke {
```

Comment:
Different focus styles for stacked and non-stacked version. Stacked version has white border around User Icon, which looks really bad when replaced with focus blue. So I'm replacing inner border with focus styling.

Reply:
So you have the same box-shadow applied on both the user-icon and the stroke?

Reply:
Yes, because stacked user-icon has white box-shadow on top of focus one which cannot be seen. So I changed stroke box-shadow to blue focus one.

Reply:
Removed box-shadow from user-icon and left only on the stroke.